fabric-command-api-v1/src/main/java/net/fabricmc/fabric/api/client/command/v1/FabricClientCommandSource.java

```diff
@@ -55,13 +58,56 @@
 	 * Gets the player that used the command.
 	 *
 	 * @return the player
+	 * @deprecated Please use {@link #getTargetPlayer()} instead.
 	 */
-	ClientPlayerEntity getPlayer();
+	@Deprecated
+	AbstractClientPlayerEntity getPlayer();
+
+	default AbstractClientPlayerEntity getTargetPlayer() {
```

Comment:
Mind copy the javadocs over from the other method

Reply:
Sure

Reply:
[This](https://github.com/FabricMC/fabric/pull/1518#discussion_r669211123) does not allow other players though, as suggested [by you](https://github.com/FabricMC/fabric/pull/1518#issuecomment-867296548). Do you want me to remove that feature?